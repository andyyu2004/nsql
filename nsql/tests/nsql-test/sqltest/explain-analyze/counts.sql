EXPLAIN ANALYZE TIMING OFF
SELECT
    *
FROM (
    VALUES (1),
        (2))
    CROSS JOIN (
        VALUES (3), (4))
