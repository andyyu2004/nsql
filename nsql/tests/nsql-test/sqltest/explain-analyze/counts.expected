 cross product in=4 out=4   
   scan 2 values in=0 out=2 
   scan 2 values in=0 out=2 
                            